src/analysis/convex/simplicial_complex/basic.lean

```diff
@@ -50,6 +50,7 @@ namespace geometry
 Note that the textbook meaning of "glue nicely" is given in
 `geometry.simplicial_complex.disjoint_or_exists_inter_eq_convex_hull`. It is mostly useless, as
 `geometry.simplicial_complex.convex_hull_inter_convex_hull` is enough for all purposes. -/
+-- TODO: update to new binder order? not sure what binder order is correct for `down_closed`.
```

Comment:
i'd just like to place some focus on this as I'm not sure, anyways this should probably be left to a follow-up